### PR TITLE
Improve docs: clarify report variable and add command line options

### PR DESCRIPTION
**Pull request by simonhoffmann**

### CODE REVIEW

File: docs/configuration/advanced.md [hunk 1]
```diff
@@ -25,6 +27,13 @@ uda.taskwarrior-tui.style.calendar.title=black on rgb444
 uda.taskwarrior-tui.task-report.next.filter=$(task show report.next.filter)
 ```
 
+The `uda.taskwarrior-tui.task-report.next.filter` variable defines the default view at program startup. Set this to any preconfigured report (`task reports`), or create your own report in taskwarrior and specify its name here.
+
+## commandline options:
+
+`-c`: specify a config file other than standard `.taskrc`
```
Comment (kdheepak) [inline docs/configuration/advanced.md:34]: Currently this doesn’t do anything, so we should remove it from the docs.

**Comment by kdheepak** [comment]
Thanks for the PR!

File: docs/configuration/advanced.md (hunk 1 above)
Comment (simonhoffmann) [reply docs/configuration/advanced.md]: I have updated the PR accordingly :)

**Comment by kdheepak** [comment]
The docs are available in this release: https://github.com/kdheepak/taskwarrior-tui/releases/tag/v0.13.34

**Pull request merged**